ui/src/components/pages/tenancy/platforms/clusters/form.js

```diff
@@ -97,7 +97,7 @@ export const ClusterForm = ({
             <Field
                 name="name"
                 label="Platform name"
-                helpText="Must contain lower-case alphanumeric characters and dash (-) only."
+                helpText="Must start with a letter, and only contain lower-case alphanumeric characters & dashes (-)."
```

Comment:
Could we specify lowercase here too?
```suggestion
                helpText="Must start with a lowercase letter, and only contain lower-case alphanumeric characters & dashes (-)."
```